msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -54,11 +54,28 @@ Create the conversation, after you have the user or channel information.
 
 ## Create the conversation
 
-Create the conversation if it doesn't exist or you don't know the `conversationId`. Create the conversation only once and store the `conversationId` value or `conversationReference` object.
+You can create the conversation if it doesn't exist or you don't know the `conversationId`. Create the conversation only once and store the `conversationId` value or `conversationReference` object.
+
+To create a conversation, you need `userId`, `tenantId`, and `serviceUrl`.
+
+For `serviceUrl`, use the value from an incoming activity triggering the flow or one of the global service URLs. The following is an example for the `serviceUrl`:
+
+```csharp
+const serviceUrl = 'https://smba.trafficmanager.net/uk/';
+        `MicrosoftAppCredentials.trustServiceUrl(serviceUrl);
+```
+
+If the `serviceUrl` isn't available from an incoming activity triggering the proactive scenario, use the following global URL endpoints:
+
+* Production: `https://smba.trafficmanager.net/teams/`
+* GCC: `https://smba.infra.gcc.teams.microsoft.com/gcc`
+* GCCH: `https://smba.infra.gov.teams.microsoft.us/gcch`
+
+Code sample to [**create a conversation bot**](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs).
```

Comment:
This file demonstrates several things that aren't related to this topic. Let's link to the specific section in the sample, or mention the method name, like "see the call the CreateConversationAsync in this sample"

Reply:
Just as an fyi, the following are also valid: 
* GCC: `https://smba.infra.gcc.teams.microsoft.com/teams`
* GCCH: `https://smba.infra.gov.teams.microsoft.us/teams`
Both /teams or /gcc will work. If  i remember correctly, using /teams would actually perform a check on the tenant region to validate it is gcc, while using /gcc directly will not. What do we prefer devs to use?
@tacri  @aosolis